src/game/bettercamera.inc.h

```diff
@@ -189,12 +189,18 @@ void newcam_diagnostics(void)
 
 static s16 newcam_adjust_value(s16 var, s16 val)
 {
-    var += val;
-    if (var > 100)
-        var = 100;
-    if (var < -100)
-        var = -100;
-
+    if (val > 0)
+    {
+        var += val;
+        if (var > max)
+            var = max;
+    }
+    else if (val < 0)
+    {
+        var += val;
+        if (var < max)
+            var = max;
+    }
```

Comment:
`error: 'max' undeclared (first use in this function); did you mean 'fmax'?`

Reply:
Fixed.